code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -458,12 +458,28 @@
 	modules += new /obj/item/device/instrument/instrument_synth(src)
 	modules += new /obj/item/weapon/tray/robotray(src)
 	modules += new /obj/item/weapon/reagent_containers/dropper/robodropper(src)
-	modules += new /obj/item/weapon/reagent_containers/glass/replenishing/cyborg(src)
+	modules += new /obj/item/weapon/reagent_containers/food/drinks/shaker(src)
+	modules += new /obj/item/weapon/reagent_containers/glass/rag/robo(src)
+	modules += new /obj/item/device/chem_synth/robot/service(src)
+	modules += new /obj/item/weapon/kitchen/utensil/knife/large(src)
+	modules += new /obj/item/weapon/kitchen/rollingpin(src)
+	modules += new /obj/item/weapon/storage/bag/food/borg(src)
 
-	emag = new /obj/item/weapon/reagent_containers/glass/replenishing/cyborg/hacked(src)
+	emag = new /obj/item/weapon/kitchen/utensil/knife/large/butch/meatcleaver(src)
 
 	fix_modules()
 
+/obj/item/weapon/robot_module/butler/on_emag()
+	//add some spicy chemicals to the synth
+	for(var/M in modules)
+		if(istype(M, /obj/item/device/chem_synth/robot/service))
+			to_chat(world, "found")
```

Comment:
might wanna message_admins() instead or just remove this

Reply:
Oh yeah, that's a debug line that I forgot in. Thanks.